doc-content/enterprise-only/project-data/preview-editor-save-test-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='preview-editor-save-test-proc']
+= Saving a test scenario
+
+You can save a test scenario at any time while creating a test scenario template or defining the test scenarios.
+
+.Procedure
+. From the *Test Scenario (Preview)* editor toolbar on the upper-right, click *Save*.
+. On the *Confirm Save* window,
+.. If you wish to add a comment regarding the test scenario, click *add a comment*.
+.. Click *Save* again.
+
+You will see a message stating that the test scenario was saved successfully.
```

Comment:
"You will see a message stating that the test scenario was saved successfully."
Same as the comment above.

Reply:
Changed as suggested.